metricbeat/module/logstash/node_stats/_meta/data.json

```diff
@@ -4,25 +4,28 @@
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
+    "event": {
+        "dataset": "logstash.node.stats",
+        "duration": 115000,
+        "module": "logstash"
+    },
     "logstash": {
         "node": {
             "stats": {
                 "events": {
-                    "filtered": 30750,
-                    "in": 1,
-                    "out": 30750
+                    "filtered": 0,
+                    "in": 0,
+                    "out": 0
                 }
             }
         }
     },
     "metricset": {
-        "host": "127.0.0.1:9600",
-        "module": "logstash",
-        "name": "node_stats",
-        "namespace": "logstash.node.stats",
-        "rtt": 115
+        "name": "node_stats"
     },
     "service": {
```

Comment:
Seems like this needs updating as the version is not here yet.

Reply:
Turned out version wasn't being parsed out of the Logstash Node Stats API response. Neither was hostname or service ID, so I added those as well to the `logstash/node_stats` metricset.

Similarly, to make things consistent, I also added service ID to the `logstash/node` metricset.